src/me/StevenLawson/TotalFreedomMod/Commands/Command_kick.java

```diff
@@ -0,0 +1,51 @@
+package me.StevenLawson.TotalFreedomMod.Commands;
+
+import me.StevenLawson.TotalFreedomMod.Commands.Command_smite;
+import me.StevenLawson.TotalFreedomMod.TFM_AdminList;
+import me.StevenLawson.TotalFreedomMod.TFM_Util;
+import org.apache.commons.lang3.ArrayUtils;
+import org.apache.commons.lang3.StringUtils;
+import org.bukkit.ChatColor;
+import org.bukkit.command.Command;
+import org.bukkit.command.CommandSender;
+import org.bukkit.entity.Player;
+
+@CommandPermissions(level = AdminLevel.SUPER, source = SourceType.BOTH)
+@CommandParameters(description = "Kick someone from the server.", usage = "/<command> <playername> [reason]")
+public class Command_kick
```

Comment:
@DUFCLiam  As stated, this PR sadly has not been tested before committing, which leads to these embarrassing oversights. Please follow the contribution guidelines from now on.